src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue

```diff
@@ -86,135 +50,80 @@
-                CUSTOM: i18n.t('DASHBOARDS.CUSTOMIZE.VARIABLES.FILTER_CUSTOM'),
-            })),
-        });
-        const overlayState = reactive({
-            contentStatus: 'LIST' as OverlayStatus,
-            titleSet: computed<Record<OverlayStatus, TranslateResult>>(() => ({
-                LIST: i18n.t('DASHBOARDS.CUSTOMIZE.VARIABLES.SUB_TITLE'),
-                ADD: i18n.t('DASHBOARDS.CUSTOMIZE.VARIABLES.ADD'),
-                EDIT: i18n.t('DASHBOARDS.CUSTOMIZE.VARIABLES.SUB_TITLE_EDIT'),
-            })),
-        });
-
-        const deleteModalState = reactive({
-            headerTitle: i18n.t('DASHBOARDS.CUSTOMIZE.VARIABLES.DELETE_TITLE'),
-            visible: false,
-            loading: false,
-        });
-
-        /* Helper */
-        const variableTypeBadgeStyleFormatter = (type: VariableType) => {
-            if (type === 'MANAGED') return 'gray';
-            return 'primary';
-        };
-        /* EVENT */
-        const handleOpenDeleteModal = () => {
-            deleteModalState.visible = true;
-        };
-        const handleSelectType = (selected) => {
-            state.selectedVariableType = selected;
-        };
-        const handleChangeAddContent = () => {
-            overlayState.contentStatus = 'ADD';
-        };
-        const handleAddVariable = () => {
-            console.log('Add Variable!');
-        };
-        const handleEditVariable = () => {
-            console.log('edit!');
-        };
-        const handleDeleteVariable = () => {
-            console.log('delete!');
-        };
-        return {
-            ...toRefs(state),
-            overlayState,
-            deleteModalState,
-            handleChangeAddContent,
-            handleOpenDeleteModal,
-            handleSelectType,
-            handleAddVariable,
-            variableTypeBadgeStyleFormatter,
-            handleEditVariable,
-            handleDeleteVariable,
-        };
-    },
+const props = defineProps<Props>();
+const emit = defineEmits<EmitFn>();
+
+const state = reactive({
+    contentType: 'LIST' as OverlayStatus,
+    titleSet: computed<Record<OverlayStatus, TranslateResult>>(() => ({
+        LIST: i18n.t('DASHBOARDS.CUSTOMIZE.VARIABLES.SUB_TITLE'),
+        ADD: i18n.t('DASHBOARDS.CUSTOMIZE.VARIABLES.ADD'),
+        EDIT: i18n.t('DASHBOARDS.CUSTOMIZE.VARIABLES.SUB_TITLE_EDIT'),
+    })),
+});
+
+const { contentType, titleSet } = toRefs(state);
+
+const deleteModalState = reactive({
+    headerTitle: i18n.t('DASHBOARDS.CUSTOMIZE.VARIABLES.DELETE_TITLE'),
+    visible: false,
+    loading: false,
+    selected: '',
 });
+
+/* EVENT */
+const handleChangeAddContent = () => {
+    state.contentType = 'ADD';
+};
+const handleOpenDeleteModal = (propertyName: string) => {
+    deleteModalState.selected = propertyName;
+    deleteModalState.visible = true;
+};
+const handleDeleteVariable = () => {
+    console.log(deleteModalState.selected, 'Delete!!!!');
```

Comment:
question: Is this temporary code, right?

Reply:
Yes!!!!!!!!!!!!!!!